apps/site/pages/id/about/index.mdx

```diff
@@ -68,4 +68,4 @@ Node.js dirancang tanpa thread bukan berarti Anda tidak dapat memanfaatkan banya
 [`child_process.fork()`]: https://nodejs.org/api/child_process.html
 [`cluster`]: https://nodejs.org/api/cluster.html
 [event machine]: https://github.com/eventmachine/eventmachine
-[berpilin]: https://twisted.org/
+[twisted]: https://twisted.org/
```

Comment:
i honestly dont know how to fix this anymore
i changed it and approved it yesterday 
![image](https://github.com/user-attachments/assets/d361b15b-f659-4e84-9f62-1c6d00bc3dfe)

Reply:
I unhid them - perhaps that will make it sync